### ISSUE TITLE
where's thresholds.npy come from?

### ISSUE THREAD

**Issue by QiaoranC** [thtang/CheXNet-with-localization#3]
Hi, i found you using different thresholds for each class, this seems interesting.
How did you generate the thresholds for each class? and why?

**Comment by thtang** [comment]
The thresholds are set to maximize the sum of True positive rate and True negative rate for each class on the validation set. In other words, sensitivity and specificity are considered here.

**Comment by QiaoranC** [comment]
Emm, not fully understand why, but thanks for the explain

**Comment by QiaoranC** [comment]
i see this `iou.oy` and `find_bbox_size.py`,  this are more like to find the best npy from a list npy.
how do i generate the `thresholds.npy`? 
Thx

**Comment by thtang** [comment]
When doing validation, compute ROC curve for each class.  `sklearn.metrics.roc_curve` could help you.
You would get three returns, fpr, tpr and thresholds. Then find out the threshold that could maximize _(tpr + (1-fpr))_ **for each class**.

**Comment by QiaoranC** [comment]
got it thx again
